calculator.rb

```diff
@@ -0,0 +1,151 @@
+#operation methods
+def add(num1, num2)
+  return num1 + num2
+end
+
+def subtract(num1, num2)
+  return num1 - num2
+end
+
+def multiply(num1, num2)
+  return num1 * num2
+end
+
+def divide(num1, num2)
+  return num1 / num2
+end
+
+def modulo(num1, num2)
+  return num1 % num2
+end
+
+def exponent(num1, num2)
+  return num1 ** num2
+end
+
+#request number methods
+def requestNum1
+  puts "We will be working with two numbers. Please enter your first number:"
+  num = gets.chomp
+  
+  return num
+end
+
+def requestNum2
+  puts "Please enter your second number:"
+  num = gets.chomp
+  
+  return num
+end
```

Comment:
`requestNum1` and `requestNum2` do almost the same thing. Is there any way you could combine them to DRY your code?